exchanges/bybit/bybit_websocket.go

```diff
@@ -0,0 +1,491 @@
+					HighPrice:  data.Kline.HighPrice,
+					LowPrice:   data.Kline.LowPrice,
+					Volume:     data.Kline.Volume,
+				}
+				return nil
+			default:
+				by.Websocket.DataHandler <- stream.UnhandledMessageWarning{Message: by.Name + stream.UnhandledMessage + string(respRaw)}
+			}
+		}
+
+		if m, ok := d["auth"]; ok {
+			log.Infof(log.WebsocketMgr, "%v received auth response: %v", by.Name, m)
+			return nil
+		}
+
+		if m, ok := d["ping"]; ok {
+			log.Infof(log.WebsocketMgr, "%v received ping: %v", by.Name, m)
+			return nil
+		}
+	case []interface{}:
+		if obj, ok := d[0].(map[string]interface{}); ok {
+			if e, ok := obj["e"].(string); ok {
+				switch e {
+				case wsAccountInfoStr:
+					var data []wsAccountInfo
+					err := json.Unmarshal(respRaw, &data)
+					if err != nil {
+						return fmt.Errorf("%v - Could not convert to outboundAccountInfo structure %s",
+							by.Name,
+							err)
+					}
+					by.Websocket.DataHandler <- data[0]
+					return nil
+				case wsOrderStr:
+					var data []wsOrderUpdate
+					err := json.Unmarshal(respRaw, &data)
+					if err != nil {
+						return fmt.Errorf("%v - Could not convert to executionReport structure %s",
+							by.Name,
+							err)
+					}
+					oType, err := order.StringToOrderType(data[0].OrderType)
+					if err != nil {
+						by.Websocket.DataHandler <- order.ClassificationError{
+							Exchange: by.Name,
+							OrderID:  data[0].OrderID,
+							Err:      err,
+						}
+					}
+					var oSide order.Side
+					oSide, err = order.StringToOrderSide(data[0].Side)
+					if err != nil {
+						by.Websocket.DataHandler <- order.ClassificationError{
+							Exchange: by.Name,
+							OrderID:  data[0].OrderID,
+							Err:      err,
+						}
+					}
+					var oStatus order.Status
+					oStatus, err = stringToOrderStatus(data[0].OrderStatus)
+					if err != nil {
+						by.Websocket.DataHandler <- order.ClassificationError{
+							Exchange: by.Name,
+							OrderID:  data[0].OrderID,
+							Err:      err,
+						}
+					}
+
+					p, err := by.extractCurrencyPair(data[0].Symbol, asset.Spot)
+					if err != nil {
+						return err
+					}
+
+					oTimeInMilliSec, err := strconv.ParseInt(data[0].OrderCreationTime, 10, 64)
+					if err != nil {
+						return err
+					}
+
+					by.Websocket.DataHandler <- &order.Detail{
+						Price:           data[0].Price,
+						Amount:          data[0].Quantity,
+						ExecutedAmount:  data[0].CumulativeFilledQuantity,
+						RemainingAmount: data[0].Quantity - data[0].CumulativeFilledQuantity,
+						Exchange:        by.Name,
+						ID:              data[0].OrderID,
+						Type:            oType,
+						Side:            oSide,
+						Status:          oStatus,
+						AssetType:       asset.Spot,
+						Date:            time.UnixMilli(oTimeInMilliSec),
+						Pair:            p,
+						ClientOrderID:   data[0].ClientOrderID,
+					}
+					return nil
+				case wsOrderFilledStr:
+					// already handled in wsOrderStr case
```

Comment:
This is not true. Please see bitmex.go `bitmexWSExecution` handling for an example. But with this case, you create an `order.Modify` with trade/fill info. This will then allow the order to be updated internally elsewhere

Reply:
Okay, I want you to help me understand this, please. 
So for `executionReport` we perform `order.Detail`
and for `ticketInfo` we perform `order.Modify`
Am I right? @gloriousCode

Reply:
Yes, you are correct!